src/compiler/transformers/classFields.ts

```diff
@@ -1074,22 +911,20 @@ namespace ts {
             if (shouldTransformPrivateElements && isPrivateIdentifier(propertyName)) {
                 const privateIdentifierInfo = accessPrivateIdentifier(propertyName);
                 if (privateIdentifierInfo) {
-                    switch (privateIdentifierInfo.placement) {
-                        case PrivateIdentifierPlacement.InstanceField: {
+                    if(!privateIdentifierInfo.kind) {

```

Comment:
I don't think we should use `kind = undefined` to indicate fields in the transformation code.

Also `if(` -> `if (`

Reply:
Changed to enum